### PR TITLE
[Ready] [Modular] Messenger Belt Bag

### PR DESCRIPTION
**Pull request by Weblure**
## About The Pull Request

Adds a selection of small messenger belt bags (fanny packs) that hold up to 8 units of mail. This can be acquired from the loadout menu (backpack tab) for 2 points and has no role restrictions.

image
image
image

## Why It's Good For The Game

As an alternative to the larger, red mail bag that can only be gotten from the cargo vendor, this belt bag gives assistants (or whoever) the option to deliver small quantities of mail to crewmembers. It also offers a different look to the bulky red mail bag.

## Changelog
:cl:
add: A new item -- the Messenger Belt Bag -- has been added to the loadout menu under the backpack tab. Comes in three varieties. Start delivering in style!
/:cl:

### CODE REVIEW

**Comment by thestubborn** [comment]
guess who missed a trailing new line

**Comment by Weblure** [comment]
> guess who missed a trailing new line

Where?

**Comment by thestubborn** [comment]
> 
> 
> > guess who missed a trailing new line
> 
> Where?

the belt file
objects/items/storage/belt

**Review by thestubborn — CHANGES_REQUESTED** [review]
this one right here gamer, just hit enter like 8 times

File: modular_skyrat/modules/customization/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+/obj/item/storage/belt/mailbelt/leather
+	name = "Leather Belt Bag"
+	desc = "A small bag with a belt, worn around the waist. It's just big enough to hold a small stack of letters. This one is made out of a synthetic leather blend."
+	icon = 'modular_skyrat/modules/customization/icons/obj/clothing/belts.dmi'
+	icon_state = "mailbelt_leather"
+	inhand_icon_state = "mailbelt_leather"
+	worn_icon = 'modular_skyrat/modules/customization/icons/mob/clothing/belt.dmi'
+	worn_icon_state = "mailbelt_leather"
+	equip_sound = 'sound/items/equip/toolbelt_equip.ogg'
```
Comment (thestubborn) [inline modular_skyrat/modules/customization/game/objects/items/storage/belt.dm]: hit enter a few times

Comment (Weblure) [reply modular_skyrat/modules/customization/game/objects/items/storage/belt.dm]: I don't understand. Are you *wanting* trailing newlines?

Comment (thestubborn) [reply modular_skyrat/modules/customization/game/objects/items/storage/belt.dm]: you need trailing new lines, the clothing code throws a fit without em

Comment (Weblure) [reply modular_skyrat/modules/customization/game/objects/items/storage/belt.dm]: Coding things for Byond is cool and fun

Thanks for the tip

File: modular_skyrat/modules/customization/game/objects/items/storage/belt.dm [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+//Byond code go brrrrrrr
+
```
Comment (Funce) [inline modular_skyrat/modules/customization/game/objects/items/storage/belt.dm]: Lots of extra Whitespace here
(Just a singular empty line will do at the end)

File: modular_skyrat/modules/customization/modules/client/loadout/backpack.dm [hunk 3]
```diff
@@ -2,10 +2,26 @@
...
+/datum/loadout_item/backpack/mailbelt_leather
+	name = "Leather Belt Bag"
+	path = /obj/item/storage/belt/mailbelt/leather
+	cost = 2
+
 /datum/loadout_item/backpack/matches
 	name = "Matchbox"
 	path = /obj/item/storage/box/matches
-
+
```
Comment (Funce) [inline modular_skyrat/modules/customization/modules/client/loadout/backpack.dm]: Whitespace again

**Review by Funce — COMMENTED** [review]
Nitpicky stuff, but looks good other than that.

File: modular_skyrat/modules/customization/modules/client/loadout/backpack.dm (hunk 3 above)
Comment (Weblure) [reply modular_skyrat/modules/customization/modules/client/loadout/backpack.dm]: It has the same amount of whitespace here as everything above and below it, though?

Comment (Weblure) [reply modular_skyrat/modules/customization/modules/client/loadout/backpack.dm]: Oh, whitespace, not line break
My brain is fishpaste

File: modular_skyrat/modules/customization/game/objects/items/storage/belt.dm (hunk 2 above)
Comment (Weblure) [reply modular_skyrat/modules/customization/game/objects/items/storage/belt.dm]: I am simple codemonkey
Someone tells me to hit enter 8 times
I hit enter 8 times

**Review by Funce — APPROVED** [review]

**Comment by Weblure** [comment]
> Your Fix/Feature pull request is currently below zero (-10). Maintainers may close future Feature/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

Why is this even a thing

**Comment by Funce** [comment]
Arbitrary points go up and down, make people fix to make funny number go up

**Comment by Weblure** [comment]
> Arbitrary points go up and down, make people fix to make funny number go up

Every time I make a feature PR, I should leave in a minor bug, and then make a second PR to fix that bug so the funny number goes up
image

**Pull request merged**